src/Article.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import "./style.css";
+import searchIcon from "./img/Search.png";
+import cardImg from "./img/image 7.png";
+
+function Article() {
+  return (
+    <div className="article">
```

Comment:
```suggestion
    <article className="article">
```
시멘틱 태그를 고려하면 이렇게 할 수 있겠죠?